### PR TITLE
 Task #4175 Update controller and view topping

### PR DESCRIPTION
**Pull request by chinn2206**
Update controller and view topping

### CODE REVIEW

File: resources/views/admin/topping_update.blade.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <li><a href="{{route('admin.index')}}">{{ __('message.title.dashboad') }}</a></li>
+    <li><a href="{{route('admin.topping.index')}}">{{ __('message.topping') }}</a></li>
+    <li class="active">{{ __('message.update') }}</li>
+@endsection
+
+@section('content')
+    <div class="card">
+        <div class="card-header">{{ __('message.topping') }}</div>
+        <div class="card-body">
+                {!! Form::open(['route' => ['admin.topping.update', $topping->id], 'method' => 'post']) !!}
```
Comment (BunnyPi04) [inline resources/views/admin/topping_update.blade.php]: tab lại

File: resources/views/admin/topping_update.blade.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
+@extends('layouts.app_admin')
+
+@section('page-title')
+    <li><a href="{{route('admin.index')}}">{{ __('message.title.dashboad') }}</a></li>
+    <li><a href="{{route('admin.topping.index')}}">{{ __('message.topping') }}</a></li>
```
Comment (BunnyPi04) [inline resources/views/admin/topping_update.blade.php]: convention

File: resources/views/admin/topping_list.blade.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                            <td>{{ $topping->price }}</td>
+                            <td>{{ $topping->quantity }}</td>
+                            <td>
+                                <a href="{{ route('admin.topping.edit', ['id' => $topping->id]) }}"
+                                   class="btn btn-outline-primary">
+                                    <i class="fa fa-edit"></i>
+                                </a>
+                                <a href="{{ route('admin.topping.destroy', ['id' => $topping->id]) }}"
+                                   class="btn btn-outline-danger">
+                                    <i  class="fa fa-trash"></i>
```
Comment (BunnyPi04) [inline resources/views/admin/topping_list.blade.php]: `<i  class`
dư 1 space

File: resources/views/admin/topping_create.blade.php [hunk 4]
```diff
@@ -0,0 +1,34 @@
+@extends('layouts.app_admin')
+
+@section('page-title')
+    <li><a href="{{route('admin.index')}}">{{ __('message.title.dashboad') }}</a></li>
+    <li><a href="{{route('admin.topping.index')}}">{{ __('message.topping') }}</a></li>
```
Comment (BunnyPi04) [inline resources/views/admin/topping_create.blade.php]: convention

**Comment by chinn2206** [comment]
ready

**Pull request merged**